week04/style.css

```
@@ -0,0 +1,96 @@
+
+
+html,body{
+    height: 99%;
+    /*왜 100%로 하면 스크롤바가 생기는지?*/
+    background-color: #c1bcea;
+}
+.top_end{
+    height: 10%;
+    /*border: 2px solid blue;*/
+}
+.background{
+    display: flex;
+    flex : 1;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    /* border: 2px solid black; */
+    height: 80%;
+    justify-content: space-around;
+    
+}
+
+.top{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    /* border: 2px solid red; */
+    height: 20%;
+    width: 30%;
+    
+    
+}
+
+.score{
+    color: white;
+    width: 40%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    /* border: 2px solid green; */
+    font-size: 45px;
+
+}
+
+
+
+.text{
+    /* border: 2px solid yellow; */
+    color: white;
+    height: 30%;
+    display: flex;
+    align-items: end;
+} 
+.middle{
+    display: flex;
+    justify-content: space-around;
+    /* border: 2px solid orange; */
+    height: 35%;
+    width: 70%;
+}
+
+/* #human_select{
+    border: 2px solid blue;
+} */
+
+/* #robot_select{
+} */
+.bottom{
+    display: flex;
+    justify-content: space-around;
+    /* border: 2px solid yellow; */
+    height: 10%;
+    width: 50%;
+}
+
+#img_toScissor,#img_toRrock,#img_toPaper{
+    width: 20%;
+    font-size: 20px;
+    border:0;
+    border-radius: 15%;
+}
+
+#reset{
+    color: white;
+    font-size: 20px;
+    border:0;
+    background-color: transparent;
+}
+
+/*가위바위보 이미지 크기*/
+#img, #robotimg{
+    width:auto;
+    height: 100%;
+}
```

불필요한 코드를 최소화하여 코드의 기능 및 가독성을 높여요!
```suggestion
body {
    width: 650px;
    margin: 0 auto;
    padding-top: 90px;
    text-align: center;
    background-color: #c1bcea;
    color: white;
}
body .result {
    padding-bottom: 35px;
}
body .result p {
    margin: 0;
}
body .result .score {
    font-size: 45px;
    padding-bottom: 10px;
}
body .select {
    display: flex;
    justify-content: space-between;
    padding-bottom: 40px;
}
body .select img {
    width: 200px;
}
body .option {
    display: flex;
    justify-content: space-around;
    padding-bottom: 40px;
}
body .option button {
    width: 120px;
    height: 50px;
    font-size: 20px;
    border: 0;
    border-radius: 15%;
}
body #reset {
    color: white;
    font-size: 20px;
    border: 0;
    background-color: transparent;
}
```